plugins/item_tasks/web_client/collections/WidgetCollection.js

```diff
@@ -22,6 +22,10 @@ var WidgetCollection = Backbone.Collection.extend({
                     params[m.id + '_girderFolderId'] = m.value().get('folderId');
                     params[m.id + '_name'] = m.value().get('name');
                     break;
+                case 'new-folder':
```

Comment:
I more or less blindly edited this file, so I'm not sure if this is correct or where this is actually used.